.prettierignore

```diff
@@ -0,0 +1,8 @@
+.nextflow*
```

Comment:
Did not know about it, need to add it to chipseq and atacseq 😍

Reply:
I can't remember adding if myself?! 😆

Reply:
It was added in the template
 https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/.prettierignore